.github/workflows/deploy-k8s.yml

```
@@ -0,0 +1,78 @@
+    - name: Deploy Service
+      run: kubectl apply -f k8s/service.yaml
+    
+    - name: Deploy ApisixRoute
+      run: kubectl apply -f k8s/apisix-route.yaml
+    
+    - name: Deploy HPA
+      run: kubectl apply -f k8s/hpa.yaml
+    
+    - name: Wait for deployment
+      run: kubectl rollout status deployment/ithaka-backend-deployment -n ${{ env.NAMESPACE }} --timeout=300s
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)